app/assets/javascripts/components/features/ui/components/onboarding_modal.jsx

```diff
@@ -25,8 +25,8 @@ const PageOne = ({ acct, domain }) => (
 
     <div>
       <h1><FormattedMessage id='onboarding.page_one.welcome' defaultMessage='Welcome to Mastodon!' /></h1>
-      <p><FormattedMessage id='onboarding.page_one.federation' defaultMessage='Mastodon is a social network that belongs to everyone.' /></p>
-      <p><FormattedMessage id='onboarding.page_one.handle' defaultMessage='You are on {domain}, one of many independent Mastodon instances. Your full handle is {handle}' values={{ domain, handle: <strong>{acct}@{domain}</strong> }}/></p>
+      <p><FormattedMessage id='onboarding.page_one.federation' defaultMessage='Mastodon is a network of independent server instances joining up to make one larger social network.' /></p>
```

Comment:
I think splitting this up into one, or multiple, sentences would be best. For example (just making this up on the fly) "Mastodon is a collection of servers which connect to form a larger network. Each server (known as an *instance*) is fully independent, blah blah federation."

Personally I feel like dropping all that terminology in a single sentence is a bit overwhelming, although I'm not clear on space requirements within the modal so maybe that's important.